kaldb/src/main/java/com/slack/kaldb/logstore/search/KaldbDistributedQueryService.java

```diff
@@ -433,4 +433,86 @@ public KaldbSearch.SearchResult doSearch(final KaldbSearch.SearchRequest request
       throw new RuntimeException(e);
     }
   }
+
+  @Override
+  public KaldbSearch.SchemaResult getSchema(KaldbSearch.SchemaRequest distribSchemaReq) {
+    // todo - this shares a significant amount of code with the distributed search request
+    //  and would benefit from refactoring the current "distributedSearch" abstraction to support
+    //  different types of requests
+
+    LOG.info("Starting distributed search for schema request: {}", distribSchemaReq);
```

Comment:
will INFO make it too noisy?

Reply:
We have the queries logged today at an INFO level, and this should be significantly lower throughput than the query endpoint for our use case.